Vipul Kumar/if else/05_grade.cpp

```diff
@@ -0,0 +1,40 @@
+#include<iostream>
+using namespace std;
+
+int main()
+{
+int marks;
+cin>>marks;
+if (marks>80)
```

Comment:
think about the case marks > 100 ? is it possible, if not so what will be the output when user give input more than 100.